app/main/helpers/common_helpers.py

```diff
@@ -0,0 +1,5 @@
+def string_to_bool(string):
```

Comment:
I don't think we need this. Python has a built in `strtobool` function:

```
>>> from distutils.util import strtobool
>>> val = "False"
>>> bool(val)
True
>>> strtobool(val)
0
>>> bool(strtobool(val))
False
```

Reply:
Sweet, thank you for that, I updated the code accordingly :)